pkg/prow/prow.go

```diff
@@ -601,8 +600,10 @@ func (o *Options) GetReleaseJobs() ([]string, error) {
 	for repo, p := range prowConfig.Postsubmits {
 		for _, q := range p {
 			for _, b := range q.Branches {
+				branch := strings.TrimLeft(b, "^")
```

Comment:
might be worth adding a comment here or extracting these lines into something like `prowBranchConfigToGitBranchName` (or something like that).

Reply:
done